network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html

```diff
@@ -0,0 +1,22 @@
+{% extends "./modular_page.html" %}
+
+{% load wagtailcore_tags mini_site_tags %}
+
+{% block pre_footer_classes %}{% if singleton_page %}col-12 col-lg-10 offset-lg-1{% else %}col-md-12{% endif %}{% endblock %}
+
+{% block subcontent %}
+<div class="cms py-4 py-md-5
+  {% if page.narrowed_page_content == True %}
+    col-lg-8 offset-lg-2
+  {% else %}
+    col-12
+  {% endif %}"
+>
+  <h1 class="h1-heading">{{ page.title }}</h1>
+  <p class="h6-heading mb-5">By {{ page.author }} | {% if page.first_published_at %}{{ page.first_published_at|date:"F j, Y" }}{% else %}not published yet{% endif %}</p>
+  <div class="blog-body">
+    {{ page.body }}
+  </div>
+  {% include "./fragments/ih_cta.html" with wrapper_class="blog-ih-cta-wrapper" %}
```

Comment:
this isn't a streamfield so maybe I should move it out of the `<div class="cms` wrapper